nixos/modules/services/networking/sickbeard.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I don't think the nixos module belongs here. Sonarr, couchpotato modules are stored under `nixos/modules/services/misc` so I think sickbeard should be put there as well.

Reply:
I think your right. Fixed.